src/source_map_utils.ts

```diff
@@ -30,7 +30,8 @@ export function extractInlineSourceMap(source: string): string {
     previousResult = result;
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
-  const base64EncodedMap = previousResult![1];
+  if (!previousResult) return null;
```

Comment:
I'm very wary of doing this - if we're mistakenly passing this function a file without a source map, we should fix that, not make this function silently accept bad input.  It's much easier to detect and fix a variable being undefined than a source map not being correct.